src/dns-test.js

```diff
@@ -166,11 +185,10 @@ async function sendUDPWebExtQuery(domain) {
     let key = "udpAWebExt";
     let errorKey = "AWebExt";
     let flags = ["bypass_cache", "disable_ipv6", "disable_trr"];
-    let randomDomain = createRandomDomain(domain);
```

Comment:
With the removal of the call to `createRandomDomain`, the function definition is not used any more. So, remove this: https://github.com/mozilla-extensions/dnssec-interference/blob/2b1833b5765e65f9137659f004ca0c7f06e01802/src/dns-test.js#L156-L169